jubilant/secrettypes.py

```diff
@@ -0,0 +1,146 @@
+"""Dataclasses that contain parsed output from juju secrets."""
+
+from __future__ import annotations
+
+import dataclasses
+import datetime
+import enum
+from typing import Any, Dict, Generic, TypeVar
+
+
+class Rotate(enum.Enum):
```

Comment:
@james-garner-canonical Do you think we should make these `(str, enum.Enum)` (equivalent of StrEnum but on versions less than Python 3.13) as plain enums are annoying to deal with with the `.value` thing?

Reply:
Actually, I think we should drop these enum types and just make them `str`. We haven't used enum types for `statustypes` fields (or anywhere else in Jubilant), and it's more flexible and future-proof. I also don't think these will be used much at all.